src/constructor.io/utils/Helpers.cs

```diff
@@ -113,6 +113,26 @@ protected static string MakeUrl(Hashtable options, List<string> paths, Hashtable
                     }
                 }
 
+                if (queryParams.Contains(Constants.FILTERS_PER_SECTION))
```

Comment:
Related to the comment I made above around the constant, I think we generally handle these abstracted fields (i.e. `FiltersPerSection` or `ResultsPerSection`) within the [AutocompleteRequest](https://github.com/Constructor-io/constructorio-netcore/blob/fd77b4c43719584abc11275e02843588863cf455/src/constructor.io/models/Autocomplete/AutocompleteRequest.cs#L109) class.

I think this way works fine as well, but curious if we should move this logic out to the AutocompleteRequest class to keep things consistent with how we handle abstracted fields. 🤔

Reply:
I think it's better this way considering how often it's reference, specifically in multiple files